backend/section/forms.py

```diff
@@ -13,16 +13,32 @@ class AddSections(forms.Form):
     name = forms.CharField(max_length=128, required=False)
     tag = forms.CharField(max_length=128, required=False)
     group = forms.CharField(max_length=128, required=False)
-    files = forms.FileField(widget=MultipleFileInput(attrs={'accept':'.wav,.mp3,.aiff,.flac,.ogg'}),
-                            validators=[audio_file_validator()])
+    files = forms.FileField(widget=MultipleFileInput(
+        attrs={'accept': '.wav,.mp3,.aiff,.flac,.ogg'}),
+        validators=[audio_file_validator()])
 
 
 class PlaylistAdminForm(forms.ModelForm):
-    csv_file = forms.FileField(required=False, help_text='Upload a CSV file (overrides the text input above)', label='CSV file', widget=forms.FileInput(attrs={'accept':'.csv'}))
+    csv_file = forms.FileField(required=False,
+                               help_text='Upload a CSV file \
+                                (overrides the text input above)',
+                               label='CSV file',
+                               widget=forms.FileInput(attrs={'accept': '.csv'})
+                               )
 
     class Meta:
         model = Playlist
         fields = '__all__'
+        help_texts = {
+            'url_prefix': 
+                'URL for hosting the audio files on an external server.<br> \
+                Make sure the folder structure matches \
+                that of the local server.<br>\
```

Comment:
I think this docstring may be really confusing in case users choose to host from a different server from the start. Perhaps you can better put "Make sure the path of the audio file is valid"?